pontoon/base/templates/project.html

```diff
@@ -24,7 +25,14 @@
 	{% if project.info_brief %}<p>{{ project.info_brief|safe }}</p>{% endif %}
 	{% if project.url %}<p><span>Website: </span><a href="{{ project.url }}">{{ project.url }}</a></p>{% endif %}
   </div>
-  {% include 'locale_selector.html' %}
+  {{ Menu.submenu((
+    (url('pontoon.project', project.slug), 'Locale', ''),
+    (url('pontoon.project.contributors', project.slug), 'Contributors', 'contributors')
+  ), request.path.split('/')[3]|default('')) }}
```

Comment:
Menu (div class="submenu") shall be moved above the form tag, to keep it consistent with contributors page. This will also fix some spacing issues.